phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/library/control/ShortcutDetailsPanel.java

```diff
@@ -63,45 +63,50 @@ public LibraryDetailsPanel(ObjectMapper objectMapper, ObjectProperty<ShortcutDTO
 
     /**
      * Constructor
-     *
-     * @param objectMapper The {@link ObjectMapper} used to load the properties from a {@link ShortcutDTO}
-     * @param shortcut The currently shown {@link ShortcutDTO} object
      */
-    public LibraryDetailsPanel(ObjectMapper objectMapper, ObjectProperty<ShortcutDTO> shortcut) {
-        this(objectMapper, shortcut, new SimpleObjectProperty<>(), new SimpleObjectProperty<>(),
-                new SimpleObjectProperty<>(), new SimpleObjectProperty<>());
+    public ShortcutDetailsPanel() {
+        this(new SimpleObjectProperty<>(), new SimpleObjectProperty<>(), new SimpleObjectProperty<>(),
+                new SimpleObjectProperty<>(), new SimpleObjectProperty<>(), new SimpleObjectProperty<>());
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
-    public LibraryDetailsPanelSkin createSkin() {
-        return new LibraryDetailsPanelSkin(this);
+    public ShortcutDetailsPanelSkin createSkin() {
+        return new ShortcutDetailsPanelSkin(this);
     }
 
     public ObjectMapper getObjectMapper() {
```

Comment:
Why do we need to wrap the object mapper inside an ObjectProperty?

Reply:
We don't need to, it is just more "clean" and flexible from a JavaFX perspective.
The idea behind using `ObjectProperty` objects is to be able to change the concrete value of the property later on without requiring to recreate the whole component.
In addition this enables me to use setter methods to specify this information without abstaining from `final` fields.

Reply:
I understand but ObjectMapper is a bean (aka a Singleton inside Phoencis). I think we should use the convention not to make bean observables, what do you think?

Reply:
That is independent from the JavaFX component perspective.
The idea is to be able to view each JavaFX component independently as a kind of sandbox.
This makes it also much easier to write tests for JavaFX components, because you don't need to setup a lot of external objects to pass to a JavaFX component.

Reply:
Ok

Reply:
After thinking about this some more I think I should rephrase my previous comment:

I agree with you that the `ObjectMapper` should be set only once, but for me there are multiple reasons to use an `ObjectProperty` for the `ObjectMapper` instance and other objects which are defined only once in Phoenicis:
- in JavaFX components object fields are represented by `ObjectProperty` objects, I want to adopt this pattern when possible in our components
- I want to define all fields in our components as `final` fields, but at the same time I don't want to pass parameters to a JavaFX component via a constructor. `ObjectProperty` objects allow this